metamath.tex

```diff
@@ -7550,12 +7564,11 @@ \subsection{Basic Syntax}
 
 %The label that begins each \texttt{\$f}, \texttt{\$e}, \texttt{\$a}, and
 %\texttt{\$p} statement must be unique.
-Each label token must be unique.
-
-No label token may match any math symbol
-token.\label{namespace}\footnote{{\em (Added June 24, 2006)} This
-restriction did not exist in earlier versions of this specification.
-While not theoretically necessary, it is imposed to make it easier to
+Each label token must be unique, and
+no label token may match any math symbol
+token.\label{namespace}\footnote{This
+restriction was added on June 24, 2006.
+It is not theoretically necessary, but is imposed to make it easier to
```

Comment:
"It is not theoretically necessary, but is imposed" - take out comma before "but"